avc/slice_test.go

```diff
@@ -65,3 +65,55 @@ func TestParseSliceHeader(t *testing.T) {
 		t.Error(diff)
 	}
 }
+
+func TestParseSliceHeader_TwoFrames(t *testing.T) {
+	wantedIdrHdr := SliceHeader{SliceType: SLICE_I, IDRPicID: 1, SliceQPDelta: 8, Size: 5}
+	wantedNonIdrHdr := SliceHeader{SliceType: SLICE_P, FrameNum: 1, ModificationOfPicNumsIDC: 3, SliceQPDelta: 13, Size: 5, NumRefIdxActiveOverrideFlag: true, RefPicListModificationL0Flag: true}
```

Comment:
Hmm. This line is 194 characters long, so it exceeds the maximum line length of 140 configured in .golangci.yml.
Just break it.

Reply:
missed that, fixed